meta/template/basic.lua

```diff
@@ -126,6 +126,12 @@ function loadfile(filename, mode, env) end
 ---@nodiscard
 function loadstring(text, chunkname) end
 
+---@version 5.1
+---@param flag_or_proxy   boolean | table
+---@return userdata
+---@nodiscard
+function newproxy(flag_or_proxy) end
```

Comment:
Please follow the name style of Lua, use lowercase without underline naming

Reply:
I see